### ISSUE TITLE
Any way to backport to Windows XP x86?

### ISSUE THREAD

**Issue by Strt-Pnk** [njz3/vJoy#25]
TL;DR:

As the title says, I'm guessing there's a reason development went past .NET Framework 4.0.
I've looked at porting it myself, but time and effort are something I cannot afford right now, and I really struggle with low-level stuff like this.

For nostalgia reasons, I would like to play older games but with a newer sim wheel, which puts me in this predicament:

Option 1: Buy an old wheel, which I would need to explain to my wife.
Option 2: If your efforts can be bought, I'll make a weak offer for the price of an old wheel ($128.63 USD locally). 
It's easier to hide than a physical object in the man cave. 
You'll also help out the boys in the retro sim community if you care about that.

Also, thanks for what you are doing, keeping vJoy semi alive. It seems like this is currently still the latest version I could find.

**Comment by njz3** [comment]
I do not know recent toolchain from Microsoft Visual Studio that can still target Windows XP SDK.
If you got an old version of Visual Studio (2008 or 2010?) and are able to compile yourself from source code, you may give it a try.

**Comment by code-monet** [comment]
Drive-by comment - you might have better luck trying to get those games to run on a modern machine (probably many other people trying to accomplish the same thing) than getting vJoy and new wheels to run on an old machine. I've seen a number of projects on GitHub in this direction e.g. https://github.com/elishacloud/dxwrapper.

Keep in mind that new wheels will come with new drivers, which may not install on the old machine. Without these drivers, there's a good chance the wheel won't be recognized as anything more than a generic USB device.

(I recommend closing this issue).
